docs/benefits_of_contextualized_ML.ipynb

```diff
@@ -0,0 +1,838 @@
+{
```

Comment:
<p>Explain why this defines a population model, or just use the baselines class to infer constant BNs. You could also compare the constant-context CBN and BN to verity intuition for the reader.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cnellington/Contextualized/pull/213/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>or just use the baselines class to infer constant BNs  - not clear to me what you meant here? is NOTMAD built on a baseline class? is there a "baselines" class? </p>

Reply:
<p>https://github.com/cnellington/Contextualized/tree/main/contextualized/baselines</p>

Reply:
<p>This should be addressed and/or population models need to be defined as a model which is context-invariant and therefore this approach makes sense.</p>